src/shared/src/types/project-types.ts

```diff
@@ -40,6 +40,7 @@ export interface Project extends WbsElement {
   slideDeckLink?: string;
   bomLink?: string;
   rules: string[];
+  endDate: Date;
```

Comment:
this should be an optional property because of the case where a project has no work packages (because then there's no end date!)